tests/unit/src/test/scala/tests/cli/MdocPropertiesSuite.scala

```diff
@@ -3,13 +3,12 @@ package tests.cli
 import mdoc.internal.cli.MdocProperties
 import munit.FunSuite
 import scala.meta.internal.io.PathIO
-import mdoc.internal.BuildInfo
 
 class MdocPropertiesSuite extends FunSuite {
   test("default") {
     val obtained = MdocProperties.default(PathIO.workingDirectory).scalacOptions
 
-    if (BuildInfo.scalaVersion.startsWith("3.0"))
+    if (mdoc.internal.BuildInfo.scalaVersion.startsWith("3.0"))
```

Comment:
FYI, I've found it very helpful to create custom MUnit tags to skip tests based on the Scala version. See https://github.com/scalameta/munit/blob/35d868403aca0e71b31b8a794472ddcbee370334/tests/shared/src/test/scala/munit/BaseSuite.scala for an example of how it's used in the MUnit codebase